app/messaging/message-request-schema.js

```diff
@@ -1,11 +1,9 @@
 import { inboundMessageSchema } from '../schemas/index.js'
 
 export const validateMessageRequest = (logger, event) => {
-  const validate = inboundMessageSchema.validate(event)
-  if (validate.error) {
-    logger.error(
-      `Message request validation error, message: ${validate.error}`
-    )
+  const { error } = inboundMessageSchema.validate(event, { abortEarly: false })
+  if (error) {
+    logger.error(`Inbound message validation error, with message: ${error}`)
```

Comment:
This will attempt to stringify the error object and end up outputting [object][object] in the logs. Better to use Pino's built in handling of errors & objects (which is also heavily optimised).

In other services we've tended to just add these info to the request logger context with setBindings rather than make multiple log entries for a single request. Maybe one to discuss with the wider team but it's the pattern we are applying elsewhere.

```suggestion
    logger.error({ err: error }, 'Inbound message validation error')
```
or even just
```suggestion
    logger.setBindings({ err: error })
```

Using `err` as that is what the Pino instance is configured to serialise.

Reply:
Thanks Matt, gone for option one for now.